src/utils.spec.ts

```diff
@@ -0,0 +1,19 @@
+import { concatAndDeDuplicate } from './utils';
+import first from './mock/abi/first.json';
+import second from './mock/abi/second.json';
+import third from './mock/abi/third.json';
+import aggregated from './mock/abi/aggregated.json';
+import { expect } from '@oclif/test';
+
+describe('utils-test', (): void => {
+  it('aggregate abis when the events of contract are separated in several files',
```

Comment:
Please wrap this `it` test case with `describe('concatAndDeDuplicate()', (): void => {})`.